src/annotator/guest.js

```diff
@@ -500,36 +515,13 @@ export default class Guest {
     return annotation;
   }
 
-  /**
-   * Open the sidebar and set the selection to `annotations` (ie. Filter the annotation
-   * list to show only these annotations).
-   *
-   * @param {AnnotationData[]} annotations
-   */
-  showAnnotations(annotations) {
```

Comment:
This method and the one below have been folded into `selectAnnotations` as these methods were not used externally.